tox.ini

```diff
@@ -28,3 +29,13 @@ basepython = python2.7
 deps =
     flake8
 commands = flake8 .
+
+[testenv:codecov]
```

Comment:
a comment about what this actually does? I'm guessing it builds some sort of report based on coverage data previously emitted?

Reply:
Done